x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/discover/search_filter.cy.ts

```diff
@@ -33,20 +33,17 @@ const INITIAL_END_DATE = 'Jan 19, 2024 @ 20:33:29.186';
 const NEW_START_DATE = 'Jan 18, 2023 @ 20:33:29.186';
 const esqlQuery = 'from auditbeat-* | where ecs.version == "8.0.0"';
 
-// Failing: See https://github.com/elastic/kibana/issues/167186
-// TODO: reuse or remove this tests when ESQL tab will be added
-describe.skip(
+describe(
   'Basic discover search and filter operations',
```

Comment:
We should rename this test as well as some locators to be aligned with the new reality.

Reply:
Done